packages/@internationalized/string/test/LocalizedStringDictionary.test.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {LocalizedStringDictionary} from '..';
+
+describe('LocalizedStringDictionary', function () {
```

Comment:
Thanks so much for adding tests!

Reply:
Maybe you are already aware of this, but I noticed that when I change extension of test file to ts, build fails because of segfault.
That was somewhat confusing, because there are ts test files in spectrum repo and for them everything works well.